### DISCUSSION TITLE
Are there any good guides on building the Alexa Skill?

### DISCUSSION THREAD

**Discussion by nbanderso** [babybuddy/babybuddy#623] [General]
Hello,

I have been dying to get the Baby Buddy Alexa skill up and running but the information provided isn't very clear.  

Does anyone know of a guide or tutorial that would help me get up to speed so that I can wrap my head around how I can incorporate this?  

Thanks in advance.

**Comment by cdubz** [comment]
Hm all we really have right now is what is in the repo README: https://github.com/babybuddy/babybuddy-alexa-skill#deployment. Do you know of any example of tutorials for other Alexa Skills we could look at for comparison? I'm just not sure what a typical deployment process looks like.

**Reply by nbanderso** [reply]
"Do you know of any example of tutorials for other Alexa Skills we could look at for comparison?"

This is actually what I'm looking for.  Basically something that can help me connect the dots in your repo.  I have read the repo numerous times but I'm not sure where to start.  

For example: 

1. Create skill.json, ask-resources.json, and ask-states.json

Maybe I'm venturing beyond my pay grade but I have no idea what this means.

**Reply by cdubz** [reply]
Haha yeah, I get it. The current documentation is developer-focused. What I'm wondering is whether you have examples of _other_ Alexa Skills you've used or deployed in the past that have more consumer friendly documentation? I don't use Alexa myself so I'm not familiar with the ecosystem.
